ionic-angular-template/src/auth/reset-password/reset-password.component.html

```diff
@@ -0,0 +1,59 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-title>Reset password</ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content class="ion-padding">
+    <ion-grid>
+      <ion-row>
+        <ion-col size-sm="6" offset-sm="3">
+          <form [formGroup]="resetPasswordForm" class="auth-form">
+            <ion-list>
+              <ion-item>
+                <ion-label position="floating">New Password</ion-label>
+                <ion-input
+                  type="password"
+                  class="form-control"
+                  formControlName='newPassword'
+                ></ion-input>
+              </ion-item>
+              <ion-item
+                *ngIf="newPassword.touched && newPassword.invalid"
+                lines="none"
+              >
+                <ion-label>
+                  Should at least be 6 characters long.
+                </ion-label>
+              </ion-item>
+              <ion-item class="item">
+                <ion-label position="floating">Confirm Password</ion-label>
+                <ion-input
+                  type="password"
+                  class="form-control"
+                  formControlName='confirmPassword'
+                ></ion-input>
+              </ion-item>
+              <ion-item
+                *ngIf="confirmPassword.touched && resetPasswordForm.hasError('notSame')"
+                lines="none"
+              >
+                <ion-label>
+                  Passwords do not match.
+                </ion-label>
+              </ion-item>
+            </ion-list>
+            <ion-button
+            class="auth-button"
+            color="primary"
+            expand="block"
+            [disabled]="!resetPasswordForm.valid"
```

Comment:
!resetPasswordForm.valid = resetPasswordForm.invalid.
We already discussed that in previous pull requests.